### ISSUE TITLE
Is there any "Artifact" issues during finetuning?

### ISSUE THREAD

**Issue by mhh0318** [cccntu/fine-tune-models#3]
I've implemented PatchGAN and StyleGAN Loss for decoder only finetuned, but I got artifacts reconstructions even after 50k steps.

My Loss function setting is: `0.1 LPIPS + 0.1 PatchGAN (with adaptive weight) / 0.1 StyleGAN (with 1e9 gradient panelty) + MSE Loss`
The learning part is `post_quant_conv`, `decoder` and `GAN` part.

The images are reconstructed without EMA. Here are some failure cases
![reconstructions_00088697280_b-001760](https://github.com/cccntu/fine-tune-models/assets/42776955/a76fc576-1602-4577-acf7-2c20807b3a7a)
![reconstructions_00083321728_b-001480](https://github.com/cccntu/fine-tune-models/assets/42776955/ca5ad44a-7384-4690-b518-335883dffbf6)
